util/keyfile/src/lib.rs

```diff
@@ -66,6 +68,10 @@ fn prost_to_io_error(err: mc_util_serial::DecodeError) -> std::io::Error {
     std::io::Error::new(std::io::ErrorKind::Other, Box::new(ProstError { err }))
 }
 
+fn hex_to_io_error(err: FromHexError) -> std::io::Error {
+    std::io::Error::new(std::io::ErrorKind::Other, Box::new(err))
+}
```

Comment:
Stuff like this, and the functions above it (`to_io_error`, `prost_to_io_error`, etc) and the `ProstError` type below mean we should just replace this with a `KeyFileError` that implements `From<IoError>`, `From<DecodeError>`, and `From<FromHexError>`, like we do literally everywhere else :-)